dev/docker-compose.yml

```
@@ -108,7 +108,7 @@ services:
     restart: on-failure:3
     command: [server, --automigrate]
     environment:
-      DEVICECONFIG_INVENTORY_URI: http://inventory:8080
+      DEVICECONFIG_INVENTORY_ADDR: http://inventory:8080
```

If I'm not mistaken, this should be `_URI`: https://github.com/mendersoftware/deviceconfig/blob/master/config/config.go#L59
```suggestion
      DEVICECONFIG_INVENTORY_URI: http://inventory:8080
```